openmc/material.py

```diff
@@ -311,6 +327,65 @@ def remove_nuclide(self, nuclide):
         if nuclide._name in self._nuclides:
             del self._nuclides[nuclide._name]
 
+    def add_macroscopic(self, macroscopic):
```

Comment:
Yeah, that makes sense that you were trying to "mirror" the `Nuclide` and `Element` adder methods. If you keep `add_macroscopic(...)` I think you should add a `remove_macroscopic(...)` (though this would be more straightforward with a `macroscopic` property setter). It can be useful to design the Python API to permit multiple simulations to be run (input files generated) in the same set of scripts - *i.e.*, interchangeability between `Macroscopic` objects in the same `Material(s)` could be useful.

At the end of the day, listen to @samuelshaner on this PR more than I since he is actually using this code already!